rules/k8s-audit-logs-enabled-native-cis/raw.rego

```diff
@@ -0,0 +1,19 @@
+package armo_builtins
+
+# CIS 3.2.1 https://workbench.cisecurity.org/sections/1126657/recommendations/1838582
+deny[msga] {
+	apiserverpod := input[_]
```

Comment:
Please add validation that this is the API Server pod, as well as a `filter.rego` file